integration/src/test/resources/ammonite/integration/compilationErrorLineNumberTest1.scala

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Could you give these test cases slightly more descriptive names? e.g. "CompileErrorDefinition.scala", "CompileErrorWithSyntheticRes.scala" etc.